### PR TITLE
Bumping to 1.0.0-pre2.

### PR DESCRIPTION
**Pull request by nicolasnoble**

### CODE REVIEW

File: src/csharp/build_packages.bat [hunk 1]
```diff
@@ -30,7 +30,7 @@
 @rem Builds gRPC NuGet packages
 
 @rem Current package versions
-set VERSION=1.0.0-pre1
+set VERSION=1.0.0-pre2
 set PROTOBUF_VERSION=3.0.0-beta3
```
Comment (hsaliak) [inline src/csharp/build_packages.bat]: Should we wait/or to bump protobuf versions to 3.0.0 before cutting pre2?

Comment (nicolasnoble) [inline src/csharp/build_packages.bat]: These should be separate pull requests. See #7577, #7578 and #7583 for instance.

Comment (nicolasnoble) [inline src/csharp/build_packages.bat]: But yes, we should wait before releasing pre2 that everything has been published. Doesn't affect the version bump however :)

Comment (hsaliak) [inline src/csharp/build_packages.bat]: Ack

**Comment by hsaliak** [comment]
LGTM
